pyscriptjs/Makefile

```diff
@@ -86,20 +86,26 @@ run-examples: setup build examples
 	make dev
 
 test:
-	make examples
 	make test-ts
 	make test-py
 	make test-integration-parallel
+	make test-examples
 
+# run all integration tests *including examples* sequentially
 test-integration:
-	make examples
 	mkdir -p test_results
 	$(PYTEST_EXE) -vv $(ARGS) tests/integration/ --log-cli-level=warning --junitxml=test_results/integration.xml
 
+# run all integration tests *except examples* in parallel (examples use too much memory)
 test-integration-parallel:
+	mkdir -p test_results
+	$(PYTEST_EXE) --numprocesses auto -vv $(ARGS) tests/integration/ --log-cli-level=warning --junitxml=test_results/integration.xml -k 'not zz_examples'
```

Comment:
I wonder if it would be more convenient to move the examples tests into their own folder. Maybe we could split this change out into it's own pr since it is very important and could use a bit of tweaking.

Reply:
Responded to this below, but don't have the idea of moving examples into their own folder, as well as holistically rethinking how  we test examples.